mk/px4_targets.mk

```diff
@@ -125,6 +125,11 @@ px4-cleandep: clean
 	$(v) find $(UAVCAN_DIRECTORY) -type f -name '*.d' | xargs rm -f
 	$(v) find $(SKETCHBOOK)/$(SKETCH) -type f -name '*.d' | xargs rm -f
 
+px4-v2-upload-solo: px4-v2
+	scp $(SKETCH)-v2.px4 root@10.1.1.10:/tmp/
```

Comment:
As is, this would allow a user to upload ArduPlane or ArduRover to a Solo.  That's probably not good.